### ISSUE TITLE
Glacier Biome generation issues

### ISSUE THREAD

**Issue by Areolata7** [McJty/LostCities#281]
I run WildNature and have successfully generated cities along with specific biomes. However I have been trying to create a very desolate world as a survival challenge. I really want to use Glacier biome but I cannot get it to work. With every attempt I select the "buffet" option and choose only Glacier. In the lost cities I choose Ancient, turn off spawners, and change the rarity of the cities from .01 to .002. All the other settings are default.

Attempts:
1. Chose Wildnature to generate, never got past loading screen. It created a save file but it would not load after I quit and restarted so I deleted it and tried again.

2. Chose Wildnature to generate, it loaded the world (very slow), seemed fine but when I found a nearby city it was cut in half and the end of the world was there (only about 12-16 chunks of the world generated around spawn and then just empty white beyond that). I backed out and reloaded twice but no more of the world would generate. I deleted the save file and tried again.

3. Chose world generator "surface" and it went fast, and the world seemed perfect. I ran for nearly the full MC day before I found a city and all was well. However upon entering the city almost everything stopped functioning, day/night still passed but I couldn't get anything from breaking objects and the 3 goats that had spawned stopped moving. I could not save/quit so I had to force quit java in task manager. Every time I reloaded I was on the very edge of the city and every time I entered it the same thing happened. I also found another city nearby and it had the exact same affect upon entering that city. I did check my randomtickspeed and it was correctly set at 3. As you can see it generated just fine (the world didn't abruptly end like other attempts). 
![2020-03-29_05 14 49](https://user-images.githubusercontent.com/56558824/77848629-b76d2e00-718b-11ea-84a8-4a8b18e5afe2.png)

4. I chose world generator "surface", it went fast, and seemed fine around spawn but had the exact same issue as attempt #2  (world ended abruptly) - 
![2020-03-29_06 28 19](https://user-images.githubusercontent.com/56558824/77848617-a02e4080-718b-11ea-8fe3-fc141e9a7eb0.png)

I run 1.14.4 and forge 28.2.0 and the most recent wildnature and lost cities. I attached my list of mods. Again I have created other worlds, without issue, using Wildnature as the generator (with Lost Cities);  specifically I chose Bialowieza_marsh, Tundra, and Prehistoric biomes. All had the same exact mods. It is only the Glacier biome that is causing me grief. Also MC never crashes despite these issues (so there are no crash reports to review). I always have to back out and force quit Java when I notice the problem. 
[Mods List.txt](https://github.com/McJtyMods/LostCities/files/4398822/Mods.List.txt)

I did keep the file for the best attempt (#3) and I can send it if needed. You can run around the world all you want but once you enter a city the entities do not move and blocks do not behave as they should when broken. 

Please advise.

**Comment by Areolata7** [comment]
Update: I tried some other biomes since then; Himalayas, Sahara, Ice Spikes, and they all generate correctly using WildNature and Lost Cities. Still only have an issue with the Glacier biome. 

I also tried generating without shaders and my resource packs. With Glacier I still got a broken world even with graphics in vanilla mode. And I can actually jump off the edges of the world (and fall for eternity). So this isn't just a matter of the terrain being invisible, it really isn't there at all.
